sdk/iot/inc/az_iot_hub_methods.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include <az_span.h>
+#include <az_result.h>
+#include "az_iot_mqtt.h"
+#include "az_iot_hub_client.h"

```

Comment:
In our SDK, we include azure headers first, then put an empty line, and then include the standard library ones, then include _az_cfg. This way, they don't get mixed when auto-formatting sorts them.

Also, why you include other headers before the include guard, is this a merge mistake or intentional? We never had to do this, but maybe you have a good reason.

Reply:
Addressed in the new PR.